src/AutoRest.CSharp.V3/Pipeline/Extensions.cs

```diff
@@ -58,15 +59,33 @@ internal static class Extensions
             _ => null
         };
 
-        private static readonly Func<string, bool, string?> NumberSerializer =
-            (vn, nu) => $"writer.WriteNumberValue({vn}{(nu ? ".Value" : string.Empty)});";
-        private static Func<string, bool, string?> StringSerializer(bool includeToString = false) =>
-            (vn, nu) => $"writer.WriteStringValue({vn}{(includeToString ? ".ToString()" : string.Empty)});";
+        private static readonly Func<string, bool, SerializationFormat, string?> NumberSerializer =
+            (vn, nu, f) => $"writer.WriteNumberValue({vn}{(nu ? ".Value" : string.Empty)});";
+        private static Func<string, bool, SerializationFormat, string?> StringSerializer(bool includeToString = false) =>
+            (vn, nu, f) => $"writer.WriteStringValue({vn}{(includeToString ? ".ToString()" : string.Empty)});";
+
+        public static string? ToFormatSpecifier(this SerializationFormat format) => format switch
+        {
+            SerializationFormat.DateTimeRFC1123 => "R",
+            SerializationFormat.DateTimeISO8601 => "S",
+            SerializationFormat.Date => "D",
+            SerializationFormat.DateTimeUnix => "U",
+            _ => null
+        };
+
+        private static readonly Func<string, bool, SerializationFormat, string?> DateTimeSerializer = (vn, nu, f) =>
+        {
+            var formatSpecifier = f.ToFormatSpecifier();
+            var valueText = $"{vn}{(nu ? ".Value" : string.Empty)}";
+            var formatText = formatSpecifier != null ? $", \"{formatSpecifier}\"" : string.Empty;
+            //TODO: Hack to call Azure.Core functionality without having the context of the namespaces specified to the file this is being written to.
+            return $"Azure.Core.Utf8JsonWriterExtensions.WriteDateTimeOffsetValue(writer, {valueText}{formatText});";
```

Comment:
This is only temporary, because all this Json-specific serialization code needs to be moved into a CodeWriter. When it is, we'll have the context of namespaces/types, and without that right now, I don't know if the files have `using Azure.Core;`. For example, Model files generally don't use Azure.Core. The operation files use Azure.Core, which is why path/query/header calls to the extension methods even work right now.

Reply:
What about adding a UseNamespace() method to the writer? I think it would endup being a one-liner.

Reply:
I see you don't have the writer here. OK this is fine then